Truffle5: issue with truffle-hdwallet-provider

Hi

I'm using Truffle 5 beta 2 and the Truffle HDWallet Provider. 
For an unknown reason, I'm not able anymore to deploy my contract to Ropsten.

I have the following error:
```
$ truffle migrate --compile-all --reset --network ropsten
Error: Cannot find module 'any-promise'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at /Users/eloudsa/ChainSkills/projects/chainlist-truffle5/node_modules/truffle-hdwallet-provider/dist/webpack:/truffle-hdwallet-provider/webpack/universalModuleDefinition:3:1
    at Object.<anonymous> (/Users/eloudsa/ChainSkills/projects/chainlist-truffle5/node_modules/truffle-hdwallet-provider/dist/webpack:/truffle-hdwallet-provider/webpack/universalModuleDefinition:1:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/eloudsa/ChainSkills/projects/chainlist-truffle5/truffle-config.js:1:26)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Function.Config.load (/usr/local/lib/node_modules/truffle/build/webpack:/packages/truffle-config/index.js:387:1)
    at Function.Config.detect (/usr/local/lib/node_modules/truffle/build/webpack:/packages/truffle-config/index.js:376:1)
    at Object.run (/usr/local/lib/node_modules/truffle/build/webpack:/packages/truffle-core/lib/commands/migrate.js:165:1)
    at Command.run (/usr/local/lib/node_modules/truffle/build/webpack:/packages/truffle-core/lib/command.js:113:1)
    at Object.<anonymous> (/usr/local/lib/node_modules/truffle/build/webpack:/packages/truffle-core/cli.js:47:1)
    at __webpack_require__ (/usr/local/lib/node_modules/truffle/build/webpack:/webpack/bootstrap 1b5412eb3ddc7c8d5561:19:1)
    at /usr/local/lib/node_modules/truffle/build/webpack:/webpack/bootstrap 1b5412eb3ddc7c8d5561:65:1
    at Object.<anonymous> (/usr/local/lib/node_modules/truffle/build/cli.bundled.js:71:10)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:282:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:743:3)
Truffle v5.0.0-beta.2 (core: 5.0.0-beta.2)
Solidity v0.5.0 (solc-js)
Node v10.14.2
```
I use the following dependencies:
````
"dependencies": {
    "dotenv": "^6.2.0",
    "truffle-hdwallet-provider": "^1.0.0-web3one.4"
  }
````

I have the same problem with `Truffle v5.0.0-next.24`.

Any idea how to fix this issue?

Thanks

--- a/packages/truffle-hdwallet-provider/package.json
+++ b/packages/truffle-hdwallet-provider/package.json
@@ -26,6 +26,8 @@
   },
   "homepage": "https://github.com/trufflesuite/truffle-hdwallet-provider#readme",
   "devDependencies": {
+    "any-promise": "^1.3.0",
+    "bindings": "^1.3.1",
     "bip39": "^2.2.0",
     "ethereumjs-tx": "^1.3.7",
     "ethereumjs-util": "^5.2.0",

